### PR TITLE
Address ruff's rule PIE790 in cosmology submodule

### PR DESCRIPTION
**Pull request by hfcpeixoto**
This pull request is to address violations to Ruff's rule **PIE790**, which is listed explicitly in issue https://github.com/astropy/astropy/issues/14818. Only submodule `cosmolgy` is addressed.

The rule has been moved from global location in `.ruff.toml` to all submodules which are still not compliant with it. These violations will be fixed in following PRs dedicated to each submodule.

This is part of a series of PRs split from https://github.com/astropy/astropy/pull/15239, following suggestions by @nstarman.

Solves part of #14818.

### CODE REVIEW

File: astropy/cosmology/_io/tests/test_model.py [hunk 1]
```diff
@@ -144,7 +144,7 @@ def test_fromformat_model_subclass_partial_info(self):
         Test writing from an instance and reading from that class.
         This works with missing information.
         """
-        pass  # there's no partial information with a Model
+        # there's no partial information with a Model
```
Comment (nstarman) [inline astropy/cosmology/_io/tests/test_model.py:147]: Please move this and the other comments to the docstring. No point having a comment without any code. 🙃

**Review by nstarman — CHANGES_REQUESTED** [review]

File: astropy/cosmology/_io/tests/test_model.py (hunk 1 above)
Comment (hfcpeixoto) [reply astropy/cosmology/_io/tests/test_model.py]: Thanks for the suggestion! I have added the comments to the docstrings.

**Comment by nstarman** [comment]
We'll have to wait for an infrastructure fix then rebase to get the CI passing.

**Comment by nstarman** [comment]
Hmm. Please rebase. Other PRs are not failing.

**Comment by hfcpeixoto** [comment]
Thanks for the ping, @nstarman. Have rebased the branch on top of `main`.

**Review by nstarman — APPROVED** [review]
Thanks @hfcpeixoto!

**Pull request merged**